middlewares/contextRetriever.go

```diff
@@ -0,0 +1,37 @@
+package middlewares
+
+import (
+	"context"
+	"fmt"
+	"github.com/companieshouse/chs.go/log"
+	"net/http"
+)
+
+var (
+	callGetRequestFromHeader = getRequestIdFromHeader
+)
+
+const XRequestId = "X-Request-Id"
+
+func WithContextID(next http.Handler) http.Handler {
```

Comment:
Please add a comment on what this function is doing ? Could we change the name of the function .

Reply:
Removed the package along with the function as it is not required any more